migration/sql-files/022-resource-type-index.sql

```diff
@@ -0,0 +1,20 @@
+-- avoid the sequence scan 
+CREATE INDEX IF NOT EXISTS idx_name_rt_name ON resource_type(name);
+
+--- The ROLE_SCOPE table had a composite primary key (PK) of ( role_id,scope_id ) 
+--- As part of the following change, a new ID column is added which is made the PK.
+--- The old PK ie ( role_id,scope_id ) is being made a Unique Key.
```

Comment:
ie ?

Reply:
ie --> i.e. --> That is :D 
Sorry for the confusion.